src/views/DocumentationTopic.vue

```diff
@@ -61,7 +70,15 @@
             :isWideFormat="enableNavigator"
             :sidenavHiddenOnLarge="sidenavHiddenOnLarge"
             @toggle-sidenav="handleToggleSidenav"
-          />
+          >
+            <template v-slot:menu-items>
```

Comment:
You can write it like
```suggestion
            <template #menu-items>
```
too

Reply:
Resolved in [54a559e](https://github.com/apple/swift-docc-render/pull/396/commits/54a559ef81ba62970191dfba7f9be5fd92e3a6a6)